components/script/stylesheet_loader.rs

```diff
@@ -147,6 +150,9 @@ impl FetchResponseListener for StylesheetContext {
                 }
                 StylesheetContextSource::Import(ref import) => {
                     let import = import.read();
+                    if elem.downcast::<HTMLLinkElement>().unwrap().is_alternate() {
```

Comment:
The test pass.

But it's true that's bizarre. As ```style``` also support [alternate](https://html.spec.whatwg.org/multipage/semantics.html#the-style-element) I should also create a method ```is_alternate``` for the ```Import``` type ?